src/scripts/main.js

```diff
@@ -22,7 +22,18 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+
+    li.textContent = key;
+    ul.append(li);
+    element.append(ul);
+
+    if (data[key]) {
```

Comment:
Consider changing the condition to `typeof data[key] === 'object' && data[key] !== null` to ensure that only non-null objects are processed recursively.